mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -526,7 +526,12 @@
             }
             controls.setPlayState(!play)
         },
-
+        
+        doStop: function () {
+		    toast('Stopping current track...')
+		    mopidy.playback.stop()
+	    },
```

Comment:
These lines mix hard tabs with soft tabs. It appears that the rest of the file uses soft tabs. Would you be able to convert these to soft tabs?